### PR TITLE
Add capability to manage SNIs & Certificates

### PR DESCRIPTION
**Pull request by marceldiass**
Add capability to manage SNIs & Certificates

### CODE REVIEW

**Comment by PGBI** [comment]
Thanks @marceldiass for this contribution!

I found a few glitches:

## When creating a new certificate:
I'm offered to choose the primary ID. Which Kong doesn't allow.
capture d ecran 2017-10-04 a 10 23 46

## When creating a new sni:
Although I have already created certificates, I can't select them. The select input is empty:
capture d ecran 2017-10-04 a 10 26 03

Side note: is there a need anyway to have a form to create an SNI? If I'm not wrong, one can create an SNI by simply adding a new hostname to a certificate.

File: src/js/controllers/snis.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      $scope.total -= 1;
+      $scope.snis.forEach(function(element, index) {
+        if (element.name === $scope.current.name) {
+          $scope.snis.splice(index, 1);
+        }
+      });
+    });
+  };
+
+  $scope.searchSnis = function() {
```
Comment (PGBI) [inline src/js/controllers/snis.js]: I don't see any search box in the SNI index page. I believe this is dead code.

Comment (marceldiass) [reply src/js/controllers/snis.js]: Done. Removed it

**Comment by marceldiass** [comment]
## Nice catch.. 

Sending modifications:
- Hide Certificate ID when creating a new certificate.

- The SNI form was the way I've found to add more SNI to an existing Certificate. 
  - I've tried to use the `PUT and PATCH  /certificates` Kong API but it did nor recognize the snis field. 

**Please could you review the controller/sni.js** code. Because I'm not able to reproduce the error in the SNI form. The Certificates list are always filled here.

**Pull request merged**